src/java_bytecode/java_string_library_preprocess.cpp

```diff
@@ -2223,29 +2061,6 @@ void java_string_library_preprocesst::initialize_conversion_table()
   cprover_equivalent_to_java_assign_and_return_function
     ["java::java.lang.StringBuilder.append:(D)Ljava/lang/StringBuilder;"]=
```

Comment:
how is the sequence done for overloaded functions ? I would have expected `D` before `L...;`